addons/website/static/src/scss/website.scss

```diff
@@ -1173,6 +1173,55 @@ table.table_desc tr td {
     }
 }
 
+// Footer Templates
+.o_footer_logo img {
+    height: $spacer * 3;
+}
+
+.o_footer_showcase {
+    .o_footer_logo {
+        margin-top: $spacer;
+        margin-bottom: $spacer;
+    }
+    .o_footer_about_us {
+        margin-bottom: $spacer * 2.5;
+    }
+}
+
+.o_footer_brand {
+    &, address, p {
+        text-align: center !important;
+    }
+    &, ul, .o_footer_address, .o_footer_logo {
+        padding-top: $spacer;
+    }
+    .fa-map-marker {
+        display: none;
+    }
+    li a {
+        text-transform: uppercase;
+    }
+}
+
+.o_footer_pizza {
+    h1 a {
+        font-size: $spacer * 1.5;
+        text-transform: none;
+        color: inherit;
+    }
+}
+
+.o_footer_brand, .o_footer_family {
+    & + .o_footer_copyright div {
```

Comment:
I feel like your custom `o_footer_XXX` classes should be applied on the `footer` element which wraps the `o_footer_copyright` element. This would avoid using `+` rules in scss which are more easily broken and also less efficient/readable.

Reply:
I deleted this part that style the footer_copyright because it's not essential for the design and anyway, the copyright will soon be restyled

Reply:
(put the class on the `footer` element anyway ;))